USFMConverter/UI/ViewData.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Avalonia.Controls;
+using System.Collections.Generic;
+
+namespace USFMConverter.UI
+{
+    public class ViewData
+    {
+        public List<string> Files { get; set; } = new();
+
+        public ComboBoxItem OutputFileFormat { get; set; }
+
+        public int SelectedTextSizeIndex { get; set; } = 0; // binding allows resetting user input
+        public ComboBoxItem TextSize { get; set; }
```

Comment:
I agree with your idea but we had one problem with that. Since we use view data binding to reduces the unnecessary event handler for every input controls, there is a limitation to that. The ComboBox (dropdown) seems to have 2 useful property that binding can make sense: SelectedIndex and SelectedItem. We chose the SelectedItem to bind because index is not reliable. Thus, the value of SelectedItem is whatever object it received from the drop drown selection - ComboBoxItem in this case. We embed a value to its "Tag" property so that we can collect at the end.

And by limiting the use of this ViewData to the front end and CoreConverter.cs, it will be mapped/adapted to the appropriate values within the converter only, thus it won't leak into the back end code.

Reply:
I suppose it's a price worth paying for the automatic binding.

Reply:
@rbnswartz feel free to let us know your thought, too